src/content/documentation/develop/best-practices-for-collecting-user-data-consents.md

```diff
@@ -41,8 +41,11 @@ To create the consent flow and consent dialogs your extension needs, you should
 
 1. Does my extension collect technical or interaction data? If so, offer the user the opportunity to opt-out of this data collection, although you can always offer opt-in consent if you prefer.
 2. Does my extension collect personally identifying information? If so, get the user’s opt-in consent before collecting any of this data. Remember that personally identifying information includes technical or interaction data tagged with the user’s identity or information that can be used to identify the user, such as an URL.
+3. Does my extension exchange data with native applications using [NativeMessaging](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Native_messaging)? If so, then the data exchanged needs to be declared in the data collection consent.
```

Comment:
This seems a bit out of place here since this section is about whether an add-on collects different kinds of data, but now how.